src/app/experience/experience.component.css

```diff
@@ -4,7 +4,23 @@
 
 .about-me-text {
   font-size: 12pt;
-  color: darkgray;
+  color: black;
+}
+
+.experience-header {
+  font-size: 12pt;
+  font-style: oblique;
+  font-weight: normal;
+  text-decoration: underline;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this usage of the experimental "text-decoration" property. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=css%3Aexperimental-property-usage)